examples/pkg/apis/example3/v1/doc.go

```diff
@@ -15,5 +15,5 @@ limitations under the License.
 */
 
 // +k8s:deepcopy-gen=package,register
-// +groupName=ThirdExample
+// +groupName=example3.some.corp
```

Comment:
This was one of the major changes from before - but I can't see any reference to having a `+groupName` just being some random string being valid. Every single case in k8s has the directory name, but with some suffix...